drivers/gles3/storage/config.h

```diff
@@ -38,11 +38,7 @@
 
 // This must come first to avoid windows.h mess
```

Comment:
```suggestion
// These must come first to avoid windows.h mess.
```
Though the comment is lying here, they're not first :)